lib/phantomjs/platform.rb

```diff
@@ -9,10 +9,6 @@ def architecture
         RbConfig::CONFIG['host_cpu']
       end
 
-      def temp_path
-        ENV['TMPDIR'] || ENV['TEMP'] || '/tmp'
```

Comment:
After rereading this, I don't see my "fix" making a lot of sense since `/tmp` should always be writable.

Confirmed. The original code works for me, I don't know why it was failing the first time I tried.